warmup/summary_statistics.py

```diff
@@ -304,6 +307,9 @@ def write_latex_table(machine, all_benchs, summary, tex_file, num_splits, with_p
 
 
 def write_html_table(summary_data, html_filename):
+    assert 'json_format_version' in summary_data and summary_data['json_format_version'] == JSON_VERSION_NUMBER, \
+        'Cannot process data from old JSON formats.'
+    del summary_data['json_format_version']
```

Comment:
Same comment as earlier. [Which also means that one can't produce HTML and LaTeX output in one run!]